src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -285,7 +307,7 @@ s32 func_8092CCEC(EnDns* this, GlobalContext* globalCtx) {
 }
 
 s32 func_8092CE38(EnDns* this) {
-    static s32 D_8092DE00[] = { 8, 8, 9 };
+    static s32 D_8092DE00[] = { EN_DNS_ANIMATION_DANCE, EN_DNS_ANIMATION_DANCE, EN_DNS_ANIMATION_FLIP };
```

Comment:
What would you call it though, `sDancingAnimations` or something?

And yeah, I'm trying to avoid scope creep. I really *just* want to document the object, but I put some other stuff in the actor based on what I've done in the past. If it's more agreeable, I can rip all that stuff out and leave the actor entirely undocumented outside of the object names.

If this won't get accepted unless En_Dns itself is fully documented, then earnestly I'll just rewrite the entire tutorial with `object_giant` instead.

Reply:
I don't think we need full documentation just because the object is being documented.

Reply:
I think it's fine, then, but the person who documents the rest of the actor is going to have to be careful to update the tutorial too.

Reply:
I tried to be very selective about what I included from the actor so that someone *wouldn't* have to update the tutorial too. The only thing in there *besides* the object symbols themselves are `sAnimations` and `sEyeTextures`, which I doubt we want to change too much. Of course, if someone changes the object symbols, they'll have to update the tutorial, but that would be true regardless of how documented the actor is.